github_pr.py

```diff
@@ -75,6 +76,81 @@ def _load_issue(**args):
     return repo.get_issue(args['number'])
 
 
+def __validate_list_of_dict(list_of_dict):
```

Comment:
i see a lot of conflicting info online http://www.bogotobogo.com/python/python_private_attributes_methods.php

can you suggest which we might want?